Libraries/libcar/Sources/AttributeList.cpp

```diff
@@ -99,3 +99,24 @@ operator()(AttributeList const &attributes) const
     /* Not a great hash. */
     return attributes.count();
 }
+
+
+std::vector<uint8_t> AttributeList::
+Write(size_t count,
+    uint32_t *identifiers) const
+{
+    std::vector<uint8_t> output = std::vector<uint8_t>(sizeof(uint16_t)*count);
+    uint16_t *values = (uint16_t *) &output[0];
```

Comment:
nit: use `reinterpret_cast<uint16_t *>`. you can also use `output.data()` since C++ 11 instead of `&output[0]`